### ISSUE TITLE
Question: Ways to use flask-admin outside of flask

### ISSUE THREAD

**Issue by milutz** [flask-admin/flask-admin#521]
## TL; DR:

Assuming I have context information (url/cookies), is there a way to call into flask-admin to make it generate HTML so I could use flask-admin with a non-flask/non-werkzeug servers?

If there isn't, are any interface close? I'm completely willing to write (and push if desired) glue code.
## Background:

I'm working in [Twisted](https://twistedmatrix.com) [Cyclone](https://github.com/fiorix/cyclone) on a highly async communication-heavy "Internet of Things" project.

After standing up the communications infrastructure and getting to (what I thought was) the tying-up-loose-ends part of the project, I've grown an appreciation for how troublesome it is to write a user administration UI.

I've been looking at flask-admin with great admiration, and have built several great sample admin interfaces in it, all the time knowing I didn't have a good way to directly use it.

Becoming frustrated with my own attempts to write something I started reading the flask-admin templates to see if I could simply pull-out the good-bits, but they just showed me flash-admin is doing is a big job.

That lead me see if I could pull [flask-admin in via WSGI](https://github.com/fiorix/cyclone/issues/127), but given my application would be made much more complex if I served it up from multiple ports and the containers compatible will current infrastructre won't let me bind WSGI to the same port as my other services this angle isn't viable for me.

This lead me to try and trace the flask-admin code, which I couldn't make heads-or-tails of on its own, so backed up and have been trying to coerce [flask into rendering flask-admin via the python shell](http://stackoverflow.com/questions/23400856/coaxing-flask-werkzeug-to-render-a-flask-admin-page-from-the-python-shell) which also hasn't gone well.

Are there any interfaces (even test interfaces?) that I could link into to play with linking flask-admin into non-flask environments?  If not are there any interfaces that are close to that?  I'm certainly will to write to code, I'm just trying to avoid reinventing the wheel.  Worst case, if none of the above is reasonable, are you aware of any other db user-admin rendering libraries (like flask-admin) that might be more open to other web frameworks?
